.snippets/code/develop/toolkit/api-libraries/papi/send-tx-transfer-keep-alive.ts

```diff
@@ -0,0 +1,6 @@
+import { MultiAddress } from '@polkadot-api/descriptors';
+
+const tx: Transaction = typedApi.tx.Balances.transfer_keep_alive({
+  dest: MultiAddress.Id('INSERT_DESTINATION_ADDRESS'),
+  value: BigInt(INSERT_VALUE),
+});
```

Comment:
Has there been an issue or a roadmap item to make the code snippets in this repo compile?